st2actions/st2actions/runners/mistral/v2.py

```diff
@@ -144,7 +144,10 @@ def try_run(self, action_parameters):
         inputs = self.runner_parameters.get('context', dict())
         inputs.update(action_parameters)
 
-        endpoint = 'http://%s:%s/v1/actionexecutions' % (cfg.CONF.api.host, cfg.CONF.api.port)
+        # If host value does not start with http, then default to http://.
+        protocol = 'http' if not cfg.CONF.api.use_ssl else 'https'
```

Comment:
If we decide to use this option we also need to update `st2common.util.api.get_full_public_api_url` function.

This function is used in many other places and if we also don't update / fix it there, those other places will still be broken.

I will open a PR for that and also update this section to use this function.